RELEASE-NOTES.md

```diff
@@ -1,5 +1,74 @@
+- https://github.com/PrefectHQ/prefect/pull/11306
+
+### Enhancements
+- Add an option through the CLI and Python client to remove schedules from deployments — https://github.com/PrefectHQ/prefect/pull/11353
+- Add client methods to interact with global concurrency limit APIs — https://github.com/PrefectHQ/prefect/pull/11415
+- Make `name` optional when saving an existing block — https://github.com/PrefectHQ/prefect/pull/11592
+- Make marking a flow as a subflow in `run_deployment`  optional — https://github.com/PrefectHQ/prefect/pull/11611
+- Improve IDE support for `PrefectObjectRegistry.register_instances` decorated classes — https://github.com/PrefectHQ/prefect/pull/11617
+- Make the UI accessible via reverse proxy and add a `--no-install` flag to `prefect dev build-ui` — https://github.com/PrefectHQ/prefect/pull/11489
+- Improve UI build during `prefect server start` - https://github.com/PrefectHQ/prefect/pull/11493
+- Improve error message in `.deploy` — https://github.com/PrefectHQ/prefect/pull/11615
+
+### Fixes
+- Use default values (if any) when no run input is provided upon `resume` — https://github.com/PrefectHQ/prefect/pull/11598
+- Prevent deployments with `RRule` schedules containing `COUNT` — https://github.com/PrefectHQ/prefect/pull/11600
+- Fix flows with class-based type hints based on `from __future__ import annotations` — https://github.com/PrefectHQ/prefect/pull/11578 & https://github.com/PrefectHQ/prefect/pull/11616
+- Raise `StepExecutionError` on non-zero `run_shell_script` return code during `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/11604
+
+### Experimental
+- Enable flow runs to receive typed input from external sources — https://github.com/PrefectHQ/prefect/pull/11573
+
+### Documentation
+- Fix non-rendering link in Docker guide — https://github.com/PrefectHQ/prefect/pull/11574
+- Update deployment and flow concept docs — https://github.com/PrefectHQ/prefect/pull/11576
+- Add examples for custom triggers to automations docs — https://github.com/PrefectHQ/prefect/pull/11589
+- Add send/receive documentation to `run_input` module docstring — https://github.com/PrefectHQ/prefect/pull/11591
+- Add automations guide — https://github.com/PrefectHQ/prefect/pull/10559
+- Fix storage guide links and reference — https://github.com/PrefectHQ/prefect/pull/11602
+- Fix typo in `prefect deploy` guide — https://github.com/PrefectHQ/prefect/pull/11606
+- Fix imports in human-in-the-loop workflows guide example — https://github.com/PrefectHQ/prefect/pull/11612
+- Add missing imports to human-in-the-loop workflows example — https://github.com/PrefectHQ/prefect/pull/11614
+- Fix formatting in `prefect deploy` guide — https://github.com/PrefectHQ/prefect/pull/11562
+- Remove "Notification blocks must be pre-configured" warning from automations docs — https://github.com/PrefectHQ/prefect/pull/11569
+- Update work pools concept docs example to use correct entrypoint — https://github.com/PrefectHQ/prefect/pull/11584
```

Comment:
- Add incident, metric, and deployment status info to automations docs — https://github.com/PrefectHQ/prefect/pull/11625